### ISSUE TITLE
Is TEAMS.md up to date?

### ISSUE THREAD

**Issue by loewenstein-sap** [paketo-buildpacks/community#164]
Are all teams' maintainers and contributors correct? If not I would suggest we fix this.

### Builders Team
@ForestEckhardt @robdimsdale @mhdawson are you active maintainers?
Also, are @ryanmoran, @arjun024, @thitch97, @sophiewigmore and @jericop active contributors?

### Content Team
@ForestEckhardt @anthonydahanne are you active maintainers?
Also, are @arjun024 @dmikusa @thitch97 @sophiewigmore and @robdimsdale active contributors?

### Dependencies Team
@sophiewigmore @brayanhenao @robdimsdale are you active maintainers?
Also, are @thitch97, @arjun024, @ForestEckhardt and @joshuatcasey active contributors?

### Go Team
@ForestEckhardt @ryanmoran @robdimsdale are you active maintainers?
Also, are @arjun024, @thitch97, @Genevieve, @sophiewigmore and @TisVictress active contributors?

### Java Team
@dmikusa @pivotal-david-osullivan @anthonydahanne are you active maintainers?
Also, are @modulo11 and @c0d1ngm0nk3y active contributors?

### .NET Core Team
@ForestEckhardt @sophiewigmore are you active maintainers?
Also, are @ryanmoran, @arjun024, @thitch97 and @joshuatcasey active contributors?

### Node.js Team
@thitch97 @ryanmoran @TisVictress @mhdawson @pacostas are you active maintainers?
Also, are @ForestEckhardt, @sophiewigmore, @fg-j, @emmjohnson, @accrazed, @arjun024, @c0d1ngm0nk3y and @robdimsdale active contributors?

### PHP Team
@thitch97 @sophiewigmore @arjun024 are you active maintainers?
Also, are @ryanmoran, @ForestEckhardt, @fg-j, @robdimsdale, @joshuatcasey and @TisVictress active contributors?

### Python Team
@thitch97 @robdimsdale @joshuatcasey are you active maintainers?
Also, are @ryanmoran, @ForestEckhardt, @sophiewigmore, @emmjohnson, @arjun024 and @TisVictress active contributors?

### Ruby Team
@ryanmoran @sophiewigmore @robdimsdale are you active maintainers?
Also, re @arjun024, @thitch97, @Genevieve, @ForestEckhardt and @fg-j active contributors?

### Rust Team
@ForestEckhardt @dmikusa are you active maintainers?

### Stacks Team
@brayanhenao @sophiewigmore @robdimsdale @mhdawson are you active maintainers?
Also, are @arjun024 @ryanmoran @fg-j and @jericop active contributors?

### Tooling Team
@ryanmoran @sophiewigmore @ForestEckhardt @robdimsdale are you active maintainers?
Also, are @arjun024 @dmikusa @thitch97 and @phil9909 active contributors?

### Utilities Team
@arjun024 @thitch97 @dmikusa @pivotal-david-osullivan @ForestEckhardt are you active maintainers?
Also, is @sophiewigmore an active contributor?

### Web Servers Team
@arjun024 @thitch97 @ForestEckhardt @TisVictress are you active maintainers?
Also, are @ryanmoran @sophiewigmore @fg-j and @robdimsdale active contributors?

**Comment by anthonydahanne** [comment]
I believe I'm an active maintainer for Java and more recently (joined last month) Content

@sophiewigmore , @robdimsdale @arjun024 @ForestEckhardt  @pivotal-david-osullivan are active maintainers

@jericop @c0d1ngm0nk3y @modulo11 are active contributors

It could be that @brayanhenao @ryanmoran  @TisVictress ,@fg-j  (and maybe even @thitch97 ?) no longer are active maintainers

For the rest, I don't know

**Comment by phil9909** [comment]
I switched jobs. At my new company, we are not using CNBs and have currently no plans of changing this. We have CI/CD based on Dockerfiles and locally just use `go run .`. It's working quite well.

For sideprojects it's also Dockerfiles everywhere. But I might create a hugo buildpack. Would that be of interest for the https://github.com/paketo-communit org?

TL;DR don't expect any major contributions from my side any time soon.

**Comment by pacostas** [comment]
I'm an active maintainer in the Node.js Team, reviewing PRs, contributing, and watching daily for any errors that need attention.

**Comment by mhdawson** [comment]
Yes for the 3 teams I'm mentioned for.

**Comment by accrazed** [comment]
Heyo, I'm not an active contributor/maintainer to the projects mentioned.

**Comment by dmikusa** [comment]
👍 mine are accurate.

**Comment by jericop** [comment]
Please keep me in the list. I still plan to contribute. I'm juggling a few too many things at the moment, but should have more time soon.

**Comment by jericop** [comment]
You could add me to the list of python contributors if that is easy enough. I have contributed a few times to python buildpack.

**Comment by dmikusa** [comment]
Just noticed that we don't even have the App Monitoring Team listed in this file :( 

App Monitoring Maintainers -> @dmikusa @joshuatcasey @pivotal-david-osullivan 
App Monitoring Contributors -> (none)

**Comment by loewenstein-sap** [comment]
I started cleaning up with removing @ryanmoran in #169. As far as I know he's not been with the project for well over a year now, right?

Given [@anthonydahanne's comment](https://github.com/paketo-buildpacks/community/issues/164#issuecomment-2223626036), I will then continue with further moves. @brayanhenao, @TisVictress, @fg-j and @thitch97 - please just chime in, if you are still an active participant in the project.

**Comment by dmikusa** [comment]
It's time for a bit of spring cleaning.

1. Maintainer clean up. Proposing moving @arjun024  @joshuatcasey to contributor status and marking them as emeritus maintainers due to lack of activity.
2. Contributor clean up. Mark @genevieve and @emmjohnson as emeritus contributors due to lack of activity.
3. Purge users from the Paketo Github org if they are no longer on any teams. This should not impact access as these users have no access beyond public. This just cleans up the roster & makes spots for new contributors.

1.) is particularly important because of the way we vote. Maintainers who do not vote throw off the math. It is also important for [RFC 63 Buildpack Status](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0063-buildpack-status.md) to have accurate lists of who is actively working on buildpacks.

Changes will be made this Friday, 2025-04-04. Please reach out prior to that if there are concerns.

**Comment by dmikusa** [comment]
I did another round of updates. I believe at this point we're all up to date.

I'm going to submit a PR shortly with a script that will synchronize TEAMS.md with the actual GitHub teams. That'll keep those in sync too.

I will see if there's a way to automate clean up of TEAMS.md too. Maybe if folks haven't contributed (PR or active discussions) in 6 months then we initiate the emeritus process. That would happen on discussions so it would extend things for 6 months.

I'll see what I can come up with. Let me know if there are ideas on this. 

Thanks!

**Comment by dmikusa** [comment]
https://github.com/paketo-buildpacks/community/pull/197
